### ISSUE TITLE
include DevExpress (.config) to mapping 

### ISSUE THREAD

**Issue by StephanHartmann** [DevExpress/devextreme-angular2#57]
According to your DevExtreme documentation I have to call
`DevExpress.config({ defaultCurrency: 'EUR' });`
for setting another default currency than USD.

Is it possible to include this DevExpress object in your angular 2 mappings?

It would be great if I can import it like
`import { DevExpress} from 'devextreme-angular2';`
and use it as described.

If this is not possible, how can I use it instead (Angular 2 with TypeScript)?

**Comment by dxbykov** [comment]
Hi,

Once we fix the #16 issue, it will be possible to import the config as follows:

````js
import config from 'devextreme/core/config'
````

As a temporary workaround, you can declare the DevExpress namespace:

````js
declare var DevExpress: any;
````

and use it as follows: 

````js
DevExpress.config({ defaultCurrency: 'EUR' });
````

**Comment by kvet** [comment]
Hi! Issue #16 has been fixed in `16.2.1-alpha.1`. You can import parts of DevExtreme functionality as mentioned above:

```js
import config from 'devextreme/core/config'
```

The whole modules structure can be found [here](https://js.devexpress.com/Documentation/16_1/Guide/Common/Modularity/#DevExtreme_Modules_Structure)
